exonum/src/messages/authorisation.rs

```diff
@@ -0,0 +1,159 @@
+use std::borrow::Cow;
+
+use failure::Error;
+use serde::Serialize;
+
+use crypto::{
+    self, hash, CryptoHash, Hash, PublicKey, SecretKey, Signature,
+};
+use messages::Message;
+use storage::StorageValue;
+
+use super::protocol::{Protocol, ProtocolMessage};
+use super::PROTOCOL_MAJOR_VERSION;
+
+use encoding::serialize::encode_hex;
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+pub struct AuthorisedMessage {
```

Comment:
Better to name it as `AuthorizedMessage` and change everywhere else 's' to 'z' in this word )

Reply:
~~AuthorizedMezzage~~